docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -347,141 +251,102 @@
        "  'start': 0,\n",
        "  'numFoundExact': True,\n",
        "  'docs': [{'id': '1',\n",
-       "    'surface_form': 'near',\n",
        "    'canonical_form': '{location_distance}',\n",
        "    'type': 'semantic_function',\n",
        "    'popularity': 90,\n",
        "    'semantic_function': 'location_distance(query, position)'},\n",
        "   {'id': '5',\n",
-       "    'surface_form': 'near',\n",
        "    'canonical_form': '{text_distance}',\n",
        "    'type': 'semantic_function',\n",
        "    'popularity': 10,\n",
        "    'semantic_function': 'text_distance(query, position)'},\n",
        "   {'id': '7',\n",
-       "    'surface_form': 'top',\n",
        "    'canonical_form': '{popular}',\n",
        "    'type': 'semantic_function',\n",
        "    'popularity': 100,\n",
        "    'semantic_function': 'popularity(query, position)'},\n",
        "   {'id': '4460243',\n",
-       "    'name': 'Charlotte',\n",
        "    'canonical_form': 'Charlotte',\n",
-       "    'surface_form': 'Charlotte',\n",
+       "    'admin_area': 'NC',\n",
```

Comment:
Question for Daniel: we used to return "surface_form". I see to remember that there's code that falls back to surface form if there's no keyword or something, which wouldn't work unless we return the surface form. My memory is hazy here, though.

TODO: confirm whether we need to return surface_form and add it back to returned fields if so.

Reply:
Added surface form in response